### PR TITLE
The ART clinic number is not displayed in UgandaEMR

### PR DESCRIPTION
**Pull request by Mwanje**
Changes made
-- added a liquibase  changeSet on a metadatamapping_metadata_set_member table, here is the link to the asana card https://app.asana.com/0/1133167201254915/1164982853641441

### CODE REVIEW

File: api/src/main/resources/liquibase.xml [hunk 1]
```diff
@@ -307,82 +307,19 @@
...
-			<column name="description" value="null" />
-			<column name="sort_weight" value="null" />
-			<column name="creator" value="2" />
-			<column name="date_created" value="2020-06-10" />
-			<column name="retired" value="0" />
-			<column name="uuid" value="4f530d1a-ad21-414d-8a53-033cf6c2db1f" />
-		</insert>
+		<comment> Enabling ART clinic number to be displayed in UgandaEMR on a patient dashboard </comment>
+		<sql>
+			INSERT INTO metadatamapping_metadata_set_member (metadata_set_id, metadata_class, metadata_uuid, sort_weight, name, description, creator, date_created, changed_by, date_changed, retired, date_retired, retired_by, retire_reason, uuid)
```
Comment (ssmusoke) [inline api/src/main/resources/liquibase.xml:319]: @Mwanje use REPLACE INTO in case the value exists

File: api/src/main/resources/liquibase.xml [hunk 2]
```diff
@@ -307,82 +307,19 @@
...
+			SELECT  1,'org.openmrs.PatientIdentifierType', (SELECT patient_identifier_type.uuid FROM patient_identifier_type WHERE patient_identifier_type.uuid='e1731641-30ab-102d-86b0-7a5022ba4115') AS metadata_uuid,null, null, null, 2,NOW(), null, null, 0, null, null, null,'57c7a9df-193d-4d44-b34a-3156e6204bde';
```
Comment (ssmusoke) [inline api/src/main/resources/liquibase.xml:320]: @Mwanje Why not just select from the patient_identifier_type table and fill the rest of the scalar values - will simplify the SELECT query

File: api/src/main/resources/liquibase.xml (hunk 1 above)
Comment (Mwanje) [reply api/src/main/resources/liquibase.xml]: Done

File: api/src/main/resources/liquibase.xml [hunk 3]
```diff
@@ -307,6 +307,24 @@
...
+		</preConditions>
+		<comment> Enabling ART clinic number to be displayed in UgandaEMR on a patient dashboard </comment>
+		<sql>
+            REPLACE INTO metadatamapping_metadata_set_member (metadata_set_member_id, metadata_set_id, metadata_class, metadata_uuid, sort_weight, name, description, creator, date_created, changed_by, date_changed, retired, date_retired, retired_by, retire_reason, uuid)
+            SELECT 1, 1, 'org.openmrs.PatientIdentifierType', 'e1731641-30ab-102d-86b0-7a5022ba4115',null, null, null, 2, NOW(), null, null, 0, null, null, null,'57c7a9df-193d-4d44-b34a-3156e6204bde';
+		</sql>
+	</changeSet>
+
+
+</databaseChangeLog>
```
Comment (ssmusoke) [inline api/src/main/resources/liquibase.xml:326]: @Mwanje This databaseChangeLog tag breaks the liquibase changeset as it already exists at the end

File: api/src/main/resources/liquibase.xml [hunk 4]
```diff
@@ -322,86 +321,6 @@
 		</sql>
 	</changeSet>
 
-
-</databaseChangeLog>
```
Comment (ssmusoke) [inline api/src/main/resources/liquibase.xml]: @Mwanje you have removed the public holiday change sets, you just need to remove a single tag

File: api/src/main/resources/liquibase.xml [hunk 5]
```diff
@@ -307,82 +307,20 @@
         </insert>
     </changeSet>
 
-	<changeSet author="rmakoba" id="ugandaemr-04052020-iss-9">
-		<preConditions onFail="MARK_RAN">
```
Comment (ssmusoke) [inline api/src/main/resources/liquibase.xml:311]: @Mwanje This should be MARK_RAN since if the data exists then there is no need to run this changeset

**Pull request merged**